packages/smooth_app/lib/query/products_preload_helper.dart

```diff
@@ -1,18 +1,22 @@
 import 'dart:io';
 import 'package:openfoodfacts/openfoodfacts.dart';
 import 'package:smooth_app/database/dao_product.dart';
-import 'package:smooth_app/database/local_database.dart';
 import 'package:smooth_app/query/product_query.dart';
 
 class PreloadDataHelper {
-  PreloadDataHelper(this.localDatabase);
-  LocalDatabase localDatabase;
+  PreloadDataHelper(this.daoProduct);
 
-  Future<String> preloadData() async {
-    final DaoProduct daoProduct = DaoProduct(localDatabase);
-    final List<ProductField> fields = ProductQuery.fields;
-    fields.remove(ProductField.KNOWLEDGE_PANELS);
+  final DaoProduct daoProduct;
+
+  /// We load the top 1000 products (without knowledge panels,
```

Comment:
For the record you'll find here the correct way to write `///` comments: https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary